### DISCUSSION TITLE
Signed MSI package

### DISCUSSION THREAD

**Discussion by liormaran** [glpi-project/glpi-agent#581] [Ideas]
### Your idea

Is it possible to get a signed MSI packages?
Most MDM solutions require you to upload a signed package to their platform in order to install them later on.

Thanks in advance.

**Comment by g-bougard** [comment]
Hi @liormaran 

it was 2 years ago, but our Code-Signing SSL certificate provider failed to renew it in a reasonable time. We then decided to change the provider... and it failed too. And honestly we don't know why they failed to achieve the purchase. As this kind of certificate is not free and we don't want to pass a lot of time to resolve such kind of provider problems, we finally decided to come back to a not signed MSI package as most people installs it via GPO and this is not an issue in that case.

We are sorry this is not convenient for most MDM solutions you may want to use.

Maybe today ssl providers are better than a year ago, but this is not in our plan to try again soon.
